### ISSUE TITLE
Deprecation warnings from GitPoller

### ISSUE THREAD

**Issue by alexreinking** [buildbot/buildbot#5693]
When I import the `GitPoller` class via:

```
from buildbot.changes.gitpoller import GitPoller
```

and then check the config, I see the following `DeprecatedApiWarning`s:

```
(venv) C:\Users\Alex Reinking\PycharmProjects\build_bot>buildbot checkconfig master\master.cfg
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\build.py:25: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.build has been deprecated, consume the bui
ldbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\buildrequest.py:25: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.buildrequest has been deprecated, c
onsume the buildbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\event.py:23: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.event has been deprecated, consume the bui
ldbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\builder.py:46: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.worker has been deprecated, consume the
buildbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\buildset.py:25: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.buildset has been deprecated, consume t
he buildbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\master.py:35: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.master has been deprecated, consume the b
uildbot.data APIs
  warn_deprecated(
c:\users\alex reinking\pycharmprojects\build_bot\venv\lib\site-packages\buildbot\status\worker.py:27: DeprecatedApiWarning: [0.9.0 and later] buildbot.status.worker has been deprecated, consume the b
uildbot.data APIs
  warn_deprecated(
Config file is good!
```

It doesn't look like GitPoller is deprecated according to the documentation, but what am I supposed to use instead?

I'm using BuildBot 2.9.2 on Python 3.8.

**Comment by p12tic** [comment]
Thanks for the bug report. This is spam of extraneous deprecation warnings, your code is fine. I've fixed the problem in #5694. The fix fill be released as Buildbot 2.9.3 in the next few days.

**Comment by alexreinking** [comment]
Thanks!
